packages/libs/react-ui/src/components/Notification/Notification.stories.tsx

```diff
@@ -0,0 +1,158 @@
+    title: {
+      control: {
+        type: 'text',
+      },
+    },
+    color: {
+      options: [
+        'default',
+        ...(Object.keys(colorVariants) as (keyof typeof colorVariants)[]),
+      ],
+      control: {
+        type: 'select',
+      },
+    },
+    expanded: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    displayCloseButton: {
+      control: {
+        type: 'boolean',
+      },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<
+  {
+    selectIcon: keyof typeof SystemIcon;
+  } & INotificationProps
+>;
+
+/*
+ *👇 Render functions are a framework specific feature to allow you control on how the component renders.
+ * See https://storybook.js.org/docs/7.0/react/api/csf
+ * to learn how to use render functions.
+ */
+
+export const Primary: Story = {
+  name: 'Notification',
+  args: {
+    selectIcon: 'Information',
+    title: 'Notification title',
+    displayCloseButton: true,
+    expanded: false,
+    color: undefined,
+  },
+  render: ({
+    selectIcon,
+    title,
+    displayCloseButton: displayCloseButton,
+    expanded: expand,
+    color,
+    simplified: simple,
+  }) => {
+    const icon = SystemIcon[selectIcon];
+    return (
+      <Notification.Container
+        icon={icon}
+        simplified={simple}
+        expanded={expand}
+        color={color}
+        title={title}
+        displayCloseButton={displayCloseButton}
+      >
+        Notification text to inform users about the event that occurred!
+        <Notification.Footer>
+          <Button title="click-me">Click me!</Button>
+        </Notification.Footer>
+      </Notification.Container>
+    );
+  },
+};
+
+export const Simplified: Story = {
+  name: 'Notification - Simplified',
+  args: {
+    selectIcon: 'Information',
+    expanded: false,
+    color: undefined,
+  },
+  render: ({ selectIcon, expanded: expand, color }) => {
+    const icon = SystemIcon[selectIcon];
+    return (
+      <Notification.Container
+        icon={icon}
+        simplified={true}
+        expanded={expand}
+        color={color}
+      >
+        Notification text to inform users about the event that occurred!
+      </Notification.Container>
+    );
+  },
+};
+
+export const Header: Story = {
+  name: 'Notification - Header',
+  args: {
+    selectIcon: 'Information',
+    title: 'Notification with a header!',
+    displayCloseButton: true,
+    expanded: false,
+    color: undefined,
+  },
+  render: ({
+    selectIcon,
+    title,
+    displayCloseButton: displayCloseButton,
+    expanded: expand,
+    color,
+    simplified: simple,
+  }) => {
+    const icon = SystemIcon[selectIcon];
+
+    return (
+      <Notification.Container
+        icon={icon}
+        simplified={simple}
+        expanded={expand}
+        color={color}
+        title={title}
+        displayCloseButton={displayCloseButton}
+      >
+        <Notification.Header>
+          <SystemIcon.Link size="md" />
+        </Notification.Header>
```

Comment:
@isaozler Will there be specific header actions for all notifications, for example will there only be 3 specific options? The reason i ask is because it might make sense to move the close icon to the HeaderActions instead of having it separate and having a prop for actions if it doesn't need to be that dynamic